src/style.css

```
@@ -1 +1,64 @@
-/* Styles go here */
+body {
+  font-family: "Roboto", sans-serif;
```

```suggestion
  font-family: "Roboto", sans-serif;
```
move it to `html`, add fallback fonts